host_os.py

```diff
@@ -34,6 +35,7 @@
     'build-package': build_package,
     'release-notes': create_release_notes,
     'upgrade-versions': upgrade_versions,
+    'update-versions-readme': update_versions_readme,
```

Comment:
I don't agree this is a good option.

Keeping it a separate command makes it more flexible, in case we only want to update the README. For example, if we want to update the README when a package (not covered by `upgrade-versions`) is changed? That wouldn't work.

Refer to infrastructure/pull/28